### DISCUSSION TITLE
**Issues with GBA roms**

### DISCUSSION THREAD

**Discussion by iamhectorsosa** [OnionUI/Onion#1362] [Q&A]
**Issues with GBA roms**

Hey, folks! I've been having issues with GBA roms. I copy the roms in my /roms directory but the do not show up once I boot up my device. I read a suggestion to reinstall the GBA package (mGBA) and copy again the roms. This isn't an ideal solution because every time I want to add a new rom I have to wipe everything clean and do this process again.

Has anyone found a workaround for this or experience a similar issue?

Model number MY354
Firmeware version 202306282128
OnionOS v4.2.3

**Comment by iamhectorsosa** [comment]
Just needed to refresh the roms which seems to wipe and regenerate the cache.

**Reply by Aemiii91** [reply]
An even better method is using the "Refresh list" option in GLO (press Y in the game list). It will only refresh the given list, instead of deleting all caches.
